foundation/foundation-upgrade/src/test/resources/specifications/v1_10_0/quarkus-bom-migration.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Updates all references to the quarkus-bom and quarkus-universe-bom to use the new aissemble-quarkus-bom for managing Quarkus dependencies
```

Comment:
A: This isn't that important, especially as we aren't currently publishing test results, but generally it's good to keep the Feature name fairly short, like a title. You can expand on the purpose or detail of the feature in the description.  This makes the readout of the test a results a bit easier to navigate/understand in my experience.

Reply:
Good to know!